resource/tasks.json

```diff
@@ -3081,6 +3084,20 @@
             "StartButton2WaitTime"
         ]
     },
+    "StartButton3": {
+        "action": "ClickSelf",
+        "preDelay": 1000,
+        "roi": [
+            1000,
+            350,
+            220,
+            340
+        ],
```

Comment:
Content like this, which is similar in nature, can actually be abbreviated as `"baseTask": "StartButton2"`. However, it's fine if you don't make the change; it doesn't really matter.